### DISCUSSION TITLE
Scroll

### DISCUSSION THREAD

**Discussion by euler007** [efemkay/obsidian-modular-css-layout#22] [Ideas]
Hi,

First off thank you for this @efemkay 

I was wondering if it would be possible to add scroll for overflow in multi-columns? This would help in fixing the size of the columns as well. 

If this doesn't work with the vision you have for the library, could you please guide me as to where in the Sheet I could add it myself so as to get the feature if at all it is possible? My CSS skills are non existent.  

Thanks!

**Comment by efemkay** [comment]
Hi, yeah by design MCL wont have overflow as i meant it to be responsive. But i think it's possible to add that option with no-wrap metadata. I can explore that first.

**Reply by euler007** [reply]
Oh awesome! I completely understand your design decision.

My use case is this:
I have multiple columns and each column has its own Dataview query. The issue faced is, if there are a lot of files the columns seem to expand endlessly. While I could control the output of Dataview it's a very hacky fix coz a bunch of files I need might not show up in the query and therefore I thought about scrolling.

A workaround would be really helpful in such scenarios.

**Comment by efemkay** [comment]
implemented the option in [v0.8.0](https://github.com/efemkay/obsidian-modular-css-layout/releases/tag/0.8.0)
